tests/manage/test_create_pvc_random_storage_class.py

```diff
@@ -46,18 +46,18 @@ def teardown():
     """
     global RBD_SECRET_OBJ, CEPHFS_SECRET_OBJ
     log.info("Deleting PVC")
-    for pvc_name in PVC_OBJ:
-        assert pvc.delete_pvcs(pvc_name.name)
+    pvc_names = [pvcs.name for pvcs in PVC_OBJS]
+    assert pvc.delete_pvcs(pvc_names)
     log.info("Deleting CEPH BLOCK POOL")
-    for pool in POOL_OBJ:
-        assert helpers.delete_cephblockpool(pool.name)
+    pool_list = [pool.name for pool in POOL_OBJS]
+    assert helpers.delete_cephblockpool(pool_list)
     log.info("Deleting RBD Secret")
     RBD_SECRET_OBJ.delete()
     log.info("Deleting CEPHFS Secret")
     CEPHFS_SECRET_OBJ.delete()
     log.info("Deleting RBD Storageclass")
-    for sc in SC_RBD_OBJ:
-        assert helpers.delete_storageclass(sc.name)
+    sc_list = [sc.name for sc in SC_RBD_OBJS]
+    assert helpers.delete_storageclass(sc_list)
```

Comment:
are you sure this function can handle list?

Reply:
done fixed in new commit